package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,813 @@
+              <distance>, <angle>
+           ],
+           [ # hbond 1 from selection 2 to the bridging water 1
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           [ # hbond 2 from selection 2 to the bridging water 1
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+
+           # hbonds linking the selection 1 and selection 2 to the bridgeing water 2
+           [ # hbond 1 from selection 1 to the bridging water 2
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           [ # hbond 1 from selection 2 to the bridging water 2
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           ....
+        ],
+        [ # frame 2
+          [ ... ], [ ... ], ...
+        ],
+        ...
+    ]
+
+Using the :meth:`WaterBridgeAnalysis.generate_table` method one can reformat
+the results as a flat "normalised" table that is easier to import into a
+database or dataframe for further processing.
+:meth:`WaterBridgeAnalysis.save_table` saves the table to a pickled file. The
+table itself is a :class:`numpy.recarray`.
+
+Detection of water bridges
+---------------------------
+Water bridges are recorded if a bridging water simultaneously form two hydrogen bonds
+with selection 1 and selection 2.
+
+Hydrogen bonds are recorded based on a geometric criterion:
+
+1. The distance between acceptor and hydrogen is less than or equal to
+   `distance` (default is 3 Å).
+
+2. The angle between donor-hydrogen-acceptor is greater than or equal to
+   `angle` (default is 120º).
+
+The cut-off values `angle` and `distance` can be set as keywords to
+:class:`WaterBridgeAnalysis`.
+
+Donor and acceptor heavy atoms are detected from atom names. The current
+defaults are appropriate for the CHARMM27 and GLYCAM06 force fields as defined
+in Table `Default atom names for Water Bridge analysis`_.
+
+Hydrogen atoms bonded to a donor are searched with one of two algorithms,
+selected with the `detect_hydrogens` keyword.
+
+"distance"
+   Searches for all hydrogens (name "H*" or name "[123]H" or type "H") in the
+   same residue as the donor atom within a cut-off distance of 1.2 Å.
+
+"heuristic"
+   Looks at the next three atoms in the list of atoms following the donor and
+   selects any atom whose name matches (name "H*" or name "[123]H"). For
+
+The "distance" search is more rigorous but slower and is set as the
+default. Until release 0.7.6, only the heuristic search was implemented.
+
+.. _Default atom names for water bridge analysis:
```

Comment:
Link to hbond analysis, or you can say that hydrogen bonds are detected as described under ...

Reply:
All of the following should not duplicated!

Reply:
@orbeckst Thank you for pointing out. I have created a link.